heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -333,7 +333,11 @@ public static void setMaxSpoutPending(Map<String, Object> conf, int max) {
   }
 
   public static void setTickTupleFrequency(Map<String, Object> conf, int seconds) {
-    conf.put(Config.TOPOLOGY_TICK_TUPLE_FREQ_SECS, Integer.toString(seconds));
+    setTickTupleFrequency(conf, (long) (seconds * 1000));
```

Comment:
can we name them explicitly. like setTickTupleFreqencySec and Msec

Reply:
but that might break backwards compatibility right?

Reply:
Ok. How about the new api then. Make it explicitly say Ms?